HDSAviz/Basic_Plot_Updated.py

```diff
@@ -0,0 +1,130 @@
+"""This is a basic script for plotting an analysis file.
+
+    This script is just the first step in visualizing the data set. This file
+    comprises of the method creating an html bokeh plot and a command at the
+    end to implement the method.
+
+"""
+from collections import OrderedDict
+
+import numpy as np
+import pandas as pd
+import pdb
+
+from bokeh.plotting import figure, show, output_file
+
```

Comment:
# Important import statement : 
Please include import statement for some tools here like below.
 
from bokeh.models.tools import (BoxZoomTool, ResetTool,
                                PreviewSaveTool,
                                ResizeTool, 
                                PanTool, PolySelectTool, 
                                WheelZoomTool)

please also see [here](http://bokeh.pydata.org/en/0.10.0/docs/user_guide/tools.html#hovertool) for instructions on implementing tools like the hover tool or pinch and zoom that we were planning.